app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb

```diff
@@ -136,6 +136,7 @@ def parse_lpar_common(lpar, type)
     end
 
     parse_vm_operating_system(vm, lpar)
+    parse_vm_ip_address(lpar, hardware)
```

Comment:
Maybe rename this to `parse_vm_networks` for consistency?

Reply:
OK